test/desktop/features/scripts/build_maze_runner.sh

```diff
@@ -5,7 +5,27 @@ if [ -z "$UNITY_VERSION" ]; then
   exit 1
 fi
 
-UNITY_PATH="/Applications/Unity/Hub/Editor/$UNITY_VERSION/Unity.app/Contents/MacOS"
+
+#!/usr/bin/env bash
+
+if [ "$(uname)" == "Darwin" ]; then
+   PLATFORM="MacOS"    
+elif [ "$(expr substr $(uname -s) 1 10)" == "MINGW64_NT" ]; then
```

Comment:
I think this can be used more portably by checking for the `OS` environment variable, which should be `Windows_NT` on all windows systems that we might care about.

```suggestion
elif [ "$OS" == "Windows_NT" ]; then
```

Reply:
done and tested